ui/src/literature/components/ParentRecordInfo.jsx

```diff
@@ -5,7 +5,13 @@ import { Link } from 'react-router-dom';
 import { getRecordIdFromRef } from '../../common/utils';
 import { LITERATURE } from '../../common/routes';
 
-function ParentRecordLink({ parentRecord }) {
+function ParentRecordInfo({ parentRecord, publicationInfo }) {
+  const pageStart = publicationInfo
+    ? publicationInfo.get(0, {}).get('page_start')
+    : null;
+  const pageEnd = publicationInfo
+    ? publicationInfo.get(0, {}).get('page_end')
+    : null;
```

Comment:
you don't need the `null` imo, it will return `undefined`

Reply:
When `publicationInfo` is `undefined`, `publicationInfo.get(...)` throws error. I have to check this before assignment. If `publicationInfo` is `undefined`, `pageStart` and `pageEnd` should also be `undefined` I think.

Reply:
What I mean you could  do the following

```
const pageStart = publicationInfo.get(0, {}).get('page_start', null)
```

Reply:
It throws an error when `publicationInfo` is `undefined` since it doesn't have a `get` function.